### PR TITLE
Add the ability to mount EFS file shares

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request reverts commit a545ecdbb46beae0c9ccf8d997f0066969fae4d3, thus re-enabling the installation of the AWS EFS utils into the AMI.

## 💭 Motivation and context ##

This is something that the assessors have decided they want after all.

## 🧪 Testing ##

I have deployed the new staging AMI to our staging COOL environment and verified that it is able to mount the EFS file share as expected.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All new and existing tests pass.
- [x] Build and test a new staging AMI with these changes.

## ✅ Pre-merge checklist ##
- [x] Finalize version.
- [x] Build new production AMI with these changes.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍 👍

File: src/aws.yml [hunk 1]
```diff
@@ -12,6 +12,18 @@
...
+    # pip modules that are installed afterwards because Ansible will
+    # see that /usr/bin/python exists and use it.  In particular,
+    # Ansible will try to install python-apt during idempotence; since
+    # this package does not exist in Debian 11 (Bullseye) that causes
+    # Ansible to fail.
+    #
+    # I have created aws/efs-utils#57 to encourage the good folks at
+    # aws/efs-utils to support python3 in their packaging.  (The code
+    # itself already supports python3.)  Until they change that, this
+    # role is the last thing that Ansible installs.
```
Comment (mcdonnnj) [inline src/aws.yml:25]: Is this still true? Per [v1.29.1](https://github.com/aws/efs-utils/releases/tag/v1.29.1) they updated the Python dependency to Python 3.

**Review by mcdonnnj — APPROVED** [review]
LGTM, but I had a question about some explanatory comments.

File: src/aws.yml (hunk 1 above)
Comment (jsf9k) [reply src/aws.yml:25]: Please see cisagov/skeleton-packer#88.

**Review by mcdonnnj — APPROVED** [review]
LGTM ✔

**Pull request merged**